src/marks/difference.js

```diff
@@ -15,11 +15,11 @@ export function differenceY(
     y,
     y1,
     y2,
-    positiveColor = "#01ab63",
-    negativeColor = "#4269d0",
-    opacity = 1,
-    positiveOpacity = opacity,
-    negativeOpacity = opacity,
+    positive = "#01ab63",
+    negative = "#4269d0",
```

Comment:
These sound like booleans to me. I definitely prefer either the *Color* or *Fill* suffix.

Reply:
OK, I'd favor positiveFill/negativeFill in that case (pushing this for now).

Reply:
I think we should use positiveFillOpacity & negativeFillOpacity, too (for symmetry, and to make clear that these affect the fillOpacity and not the opacity of the constituent areas).